commands/list.js

```diff
@@ -1,4 +1,28 @@
+const dir = `${require('os').homedir()}/.terminote/`;
+const color = require('colors');
+const error = require('../util/error');
+
 module.exports = (args) => {
-  console.log('Here are your notes!');
+  const file = require(dir + args.f);
+  const { title, titleColour, colour, type, entries } = file;
+  let checkbox;
+  console.log(title[titleColour] + '\n');
+
+  if (entries.length === 0) {
```

Comment:
Could be replaced by `if (!entries.length)` because 0 is falsy. This way can be kept for clarity